### PR TITLE
Fixed typos

### LINKED ISSUE #8854: Typo
I believe there is a typo in these sentences.

"By default these accessors have the same visibility, or access level: that of the "//** no need comma or :

"Also, the accessibility level of both the property/indexer, an" //** no need for second comma

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 865bd72b-8d26-ec50-4fc0-e3754a00c00c
* Version Independent ID: 5607825a-87ff-1138-3743-0f1fd8408b16
* Content: [Restricting Accessor Accessibility (C# Programming Guide)](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/restricting-accessor-accessibility#see-also)
* Content Source: [docs/csharp/programming-guide/classes-and-structs/restricting-accessor-accessibility.md](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/restricting-accessor-accessibility.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by CalvinAllen**
## Summary

Fixed the typos (slight variation of the proposed fix for the second example)

Fixes #8854

### CODE REVIEW

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for submitting this PR, @CalvinAllen. I really like the first correction that you made. I thought that the second could be streamlined a bit, though.

**Comment by CalvinAllen** [comment]
Completely agree, that sounds a lot better!

**Comment by CalvinAllen** [comment]
Sorry, @rpetrusha , scheduling conflicts keeping me from resolving this - will get on as soon as I can.

File: docs/csharp/programming-guide/classes-and-structs/restricting-accessor-accessibility.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ The [get](../../../csharp/language-reference/keywords/get.md) and [set](../../..
 -   The accessibility level on the accessor must be more restrictive than the accessibility level on the property or indexer itself.  
   
 ## Access Modifiers on Overriding Accessors  
- When you override a property or indexer, the overridden accessors must be accessible to the overriding code. Also, the accessibility level of both the property/indexer, and that of the accessors must match the corresponding overridden property/indexer and the accessors. For example:  
+ When you override a property or indexer, the overridden accessors must be accessible to the overriding code. Also, the accessibility level of both the property/indexer, and that of the accessors, must match the corresponding overridden property/indexer and the accessors. For example:
```
[the remark this answers is not in the document]
Comment (CalvinAllen) [reply docs/csharp/programming-guide/classes-and-structs/restricting-accessor-accessibility.md]: Yeah, that sounds a lot better!

(moving comment from general comments to review)

**Comment by rpetrusha** [comment]
No worries, @CalvinAllen. Thanks for making the additional changes. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**